packages/common/src/custom-validators.ts

```diff
@@ -0,0 +1,29 @@
+import { 
+  registerDecorator, 
+  ValidationArguments, 
+  ValidationOptions 
+} from 'class-validator';
+
+export function IsEqualTo(property: string, validationOptions?: ValidationOptions) {
```

Comment:
This is a custom validator for class-validator. This will essentially ensure that two values in a dto matches.